gpswearos/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
```

Comment:
Could you please change the werable folder name to just `wear` instead of `gpswearos`?

Reply:
> Could you please change the wearable folder name to just `wear` instead of `gpswearos`?

Ok, thanks!